### PR TITLE
Update wording and format of shutting down and restarting a cluster

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

Some titles are too long and I changed them to fit in the website UI.

### CODE REVIEW

**Comment by Sherlock113** [comment]
@ruiyaoOps Please have a look at this PR. It seems that GitHub does not allow me to assign it to you.

**Review by ruiyaoOps — COMMENTED** [review]
cool

File: content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md [hunk 1]
```diff
@@ -42,32 +41,34 @@ done
 ```
 Then you can shut down other cluster dependencies, such as external storage.
 
-## Restart the Cluster Gracefully
-This is about restarting the cluster gracefully after shutting down cluster gracefully.
+## Restart Cluster Gracefully
+You can restart a cluster gracefully after shutting down the cluster gracefully.
```
Comment (FeynmanZhou) [inline content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md:45]: @ruiyaoOps  @Sherlock113 @pixiake   Never hear about the word `restart a cluster`, is it more exact to update it to `restart all nodes in the cluster`?

File: content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md [hunk 2]
```diff
@@ -1,38 +1,37 @@
 ---
-title: "Shutting Down and Restart Cluster Gracefully"
-description: "Demonstrate how to shut down and restart kubernetes cluster Gracefully"
+title: "Cluster Shutdown and Restart"
```
Comment (FeynmanZhou) [inline content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md:2]: Is it more exact to update the title to `shutting down and restart all nodes in the cluster`?

File: content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md (hunk 1 above)
Comment (Sherlock113) [reply content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md:45]: image
Please have a look at this, the [OpenShift doc](https://docs.openshift.com/container-platform/4.5/backup_and_restore/graceful-cluster-restart.html).

File: content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md (hunk 2 above)
Comment (Sherlock113) [reply content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md:2]: image
[openshift doc](https://docs.openshift.com/container-platform/4.5/backup_and_restore/graceful-cluster-shutdown.html)

**Pull request merged**